src/Utils/StrTree.dfy

```diff
@@ -67,14 +67,14 @@ module Utils.StrTree {
   lemma countFormat_Acc(formatString: string, start: int := 0, count: int := 0)
     decreases |formatString| - start
     ensures countFormat(formatString, start, count)
-         == countFormat(formatString, start) + count
+            == countFormat(formatString, start) + count
```

Comment:
I wonder how hard this one would be to support

Reply:
Not that hard because I had already implemented this alignment behavior for chaining expressions.
I now implemented it for <, <=, ==, !=, <==>, >, >=